src/components/about-us/section-02/arrows.js

```diff
@@ -0,0 +1,78 @@
+import { screen } from '../utils/screen'
+import ArrowNextIcon from '../../../../static/asset/about-us/arrow-next.svg'
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+
+const Arrow = styled.div `
+  width: 30px;
+  height: 30px;
+  position: absolute;
```

Comment:
If we want `Arrows` component to be standalone, I will suggest that this component should be in a `<div style="position: relative; width: 100%; height: 100%">`